### DISCUSSION TITLE
next/image and responsiveness

### DISCUSSION THREAD

**Discussion by jensober** [igordanchenko/yet-another-react-lightbox#84] [Q&A] [answered]
I'm using the lightbox with the next/image component as described here: https://yet-another-react-lightbox.com/examples/nextjs

I get problems when I change the size of the browser window or rotate the smartphone. Here you can see a screen capture:

![LightboxNextjsResponsive](https://user-images.githubusercontent.com/8683821/224028472-c7f87fbc-940e-49e7-98cd-561f2b5be101.gif)

Is there anything I can fix here?

**Comment by igordanchenko** [comment, marked as answer]
This definitely looks like a bug. Please follow #85 for updates.
